libraries/Microsoft.Bot.Builder.Dialogs/WaterfallDialog.cs

```diff
@@ -103,6 +124,16 @@ public override async Task<DialogTurnResult> ContinueDialogAsync(DialogContext d
             return await ResumeDialogAsync(dc, DialogReason.ContinueCalled, dc.Context.Activity.Text, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Called when a child waterfall dialog completed this turn, returning control to this dialog.
```

Comment:
'Called when a child waterfall dialog completed this turn', It could be its turn instead of this turn?

Reply:
That sounds better. Thanks!